src/cidr/v6/ipv6_cidr.rs

```diff
@@ -123,6 +119,7 @@ impl Ipv6Cidr {
     }
 
     #[allow(clippy::should_implement_trait)]
+    #[doc(hidden)] // TODO: remove and rely on std FromStr
```

Comment:
Is this function necessary to be hidden?

Reply:
removed this particular change from this PR